src/jdk.compiler/share/classes/com/sun/tools/javac/code/Types.java

```diff
@@ -1051,6 +1086,10 @@ private boolean isSubtypeUncheckedInternal(Type t, Type s, boolean capture, Warn
                 }
             } else if (isSubtype(t, s, capture)) {
                 return true;
+            } else if (allowUniversalTVars && t.hasTag(TYPEVAR) && s.hasTag(TYPEVAR) && t.tsym == s.tsym) {
```

Comment:
Are you sure the check does what the comment says?

Reply:
will remove the comment, probably a self comment I guess